airflow/cli/commands/info_command.py

```diff
@@ -155,12 +156,16 @@ class Architecture:
     ARM = "arm"
 
     @staticmethod
-    def get_current():
+    def get_current() -> Architecture:
         """Get architecture."""
-        return _MACHINE_TO_ARCHITECTURE.get(platform.machine().lower())
+        current_architecture = _MACHINE_TO_ARCHITECTURE.get(platform.machine().lower())
+        if current_architecture:
+            return current_architecture
+        else:
+            raise RuntimeError(f"Unknown architecture for machine: {platform.machine()}")
```

Comment:
Some `get_current` usages has built-in handling of the `None` case here, do we want to continue to allow that?

Reply:
Right. I changed it and added UNKNOWN returned instead and removed the conditional (It was in one place). Also I noticed that OperatingSystem was similar and I changed it into Enum as well (there we had to use "UNKNOWN" because it was more likely we will have some unhandled platform.